versioned_docs/version-stable/sdk/overview.md

```diff
@@ -4,7 +4,11 @@ description: "Overview of the current status of the Typescript SDK"
 hide_title: false 
 title: "Overview"  
 ---
- 
+
+:::note
+With the new release of [v1.1.8](https://github.com/nymtech/nym/tree/release/v1.1.8), the default setting for reply-SURBS is now enabled. This allows services such as Network Requesters to send larger files privately back to the user client.
```

Comment:
change 'the user client' to just 'users' + do this on the other version of the doc